rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2047,6 +2047,22 @@ void getLastRetiredFederationP2SHScript_afterRSKIP186_whenSavingScript_returnsSc
         assertThat(lastRetiredFederationP2SHScript.get(), is(p2shScript));
     }
 
+    @Test
+    @Tag("clear retired federation")
+    void clearRetiredFederation_whenHavingOldFederation_removesOldFederation() {
+        ErpFederation federation = new P2shErpFederationBuilder().build();
+        storageProvider.setOldFederation(federation);
+
+        // check the old federation was correctly saved
+        Federation oldFederation = storageProvider.getOldFederation(federationMainnetConstants, mock(ActivationConfig.ForBlock.class));
```

Comment:
Move the mock activations into its own variables then use it in the two places where it's been needed

Reply:
good one, thanks!

Reply:
done